routes/api/resources.js

```diff
@@ -23,10 +24,23 @@ router.get('/', async (req, res) => {
   res.json(records.map((record) => record.toJSON()));
 });
 
-router.post('/', async (req, res) => {
+router.post('/', interceptors.requireLogin, async (req, res) => {
```

Comment:
The interceptors file contains functions that I call "interceptors" because they "intercept" a request before the route handler code you've written executes. You see here how `interceptors.requireLogin` is between the path and the function in the arguments. If there isn't a logged in user, an error will be returned, and the function won't execute.